### PR TITLE
[CWS] do not modify seccomp in e2e tests

### PR DESCRIPTION
**Pull request by spikat**
### What does this PR do?

Use security-agent for reloading policies on cws e2e kubernetes tests.

### Additional Notes

The initial plan was to also use security-agent to download policy, but it will require:
- a way to check that the downloaded policy contains the needed test rule.
- a way(/trick) for kubernetes e2e tests to pass env variables to the exec.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py [hunk 1]
```diff
@@ -88,15 +75,43 @@ def test_open_signal(self):
...
-            self.kubernetes_helper.kill_agent("system-probe", "-HUP")
-            time.sleep(60)
+        with Step(msg="push policies", emoji=":envelope:"):
+            self.kubernetes_helper.push_policies(self.policies)
+
+        with Step(msg="reload policies", emoji=":rocket:"):
+            self.kubernetes_helper.reload_policies()
+
+        with Step(msg="check ruleset_loaded", emoji=":delivery_truck:"):
+            for i in range(1, 60):  # retry 60 times
```
Comment (paulcacheux) [inline test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py:105]: range(60) should be enough

File: test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py [hunk 2]
```diff
@@ -88,15 +75,43 @@ def test_open_signal(self):
...
+                self.assertNotEqual(i, 59)  # timeout
```
Comment (paulcacheux) [inline test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py:106]: Not a huge fan of this, can this be replaced with https://book.pythontips.com/en/latest/for_-_else.html#else-clause or a simpler control flow ?

**Review by paulcacheux — CHANGES_REQUESTED** [review]

File: test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py (hunk 1 above)
Comment (spikat) [reply test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py]: Applied in last commit. Thanks.

File: test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py (hunk 2 above)
Comment (spikat) [reply test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py:106]: Applied in last commit. Thanks.

File: test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py [hunk 3]
```diff
@@ -102,7 +102,7 @@ def test_open_signal(self):
             self.kubernetes_helper.reload_policies()
 
         with Step(msg="check ruleset_loaded", emoji=":delivery_truck:"):
-            for i in range(1, 60):  # retry 60 times
+            for i in range(60):  # retry 60 times
                 self.assertNotEqual(i, 59)  # timeout
```
Comment (paulcacheux) [inline test/e2e/cws-tests/tests/test_e2e_cws_kubernetes.py:106]: You can remove this as well

**Review by paulcacheux — APPROVED** [review]

**Review by safchain — APPROVED** [review]

**Pull request merged**